service/controller/aws/v2/version_bundle.go

```diff
@@ -10,17 +10,17 @@ func VersionBundle() versionbundle.Bundle {
 	return versionbundle.Bundle{
 		Changelogs: []versionbundle.Changelog{
 			{
-				Component:   "Cluster Operator",
-				Description: "TODO",
+				Component:   "cluster-operator",
```

Comment:
I changed this due to https://github.com/giantswarm/fmt/blob/master/releases/how_write_changelog_message.md#rules and common style how we write operator names.